app/models/user.rb

```diff
@@ -527,10 +527,6 @@ def receives_follower_email_notifications?
     email.present? && subscribed_to_email_follower_notifications?
   end
 
-  def hotness_score
```

Comment:
This wraps a method that was marked as ready for removal by rhymes who was one of the main people working on the ES -> PG migration.